### DISCUSSION TITLE
Question about a series of apps with the same icons

### DISCUSSION THREAD

**Discussion by maciej-klupp** [Arcticons-Team/Arcticons#1249] [Q&A] [answered]
There is a series of university apps that are used across Polish universities.

**They all use the same icon:**
![image](https://user-images.githubusercontent.com/100523238/177864852-a2c25a5a-9f20-43b2-876b-bdc69869dd36.png)
**however the component info is different for each one:**
![image](https://user-images.githubusercontent.com/100523238/177864290-a33b2e28-18f5-4364-aabb-a51a61c1b91d.png)

Since I want to make an icon for the version used by the uni I study at I want to make it available for all the other ones. How should I proceed in this situation?

**Comment by Donnnno** [comment, marked as answer]
Make one, give the rest the same drawable name :)

You can also include all component info's under the same name

**Comment by Aneeq-Tahir** [comment]
i am doing the same thing for my college but my svg file only has the svg tag and it's path tags and no xml declaration since i generated it from figma and i want to know if the xml declaration is necessary and how do i create components like above for my college apps?

**Reply by Donnnno** [reply]
Hey, take a look at the icon request app here: https://docs.arcticons.com/contribute/the-iconrequest-app
that should do the trick!
